client/src/components/Dashboard.js

```diff
@@ -18,7 +18,7 @@ const Dashboard = () => {
 
```

Comment:
something's off with the above function--you shouldn't be handling an `out` when the count gets to 4 balls, it'd result in a walk and a runner on base. instead, the app looks like it increments the `innings` count when ball 4 occurs